### PR TITLE
Add parameter -i to specify the key file used to connect to the target machine

### PR DESCRIPTION
**Pull request by Oreoxmt**
### What is changed, added or deleted? (Required)
Add parameter -i to specify the key file used to connect to the target machine when `tiup cluster deploy`.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [ ] v6.1 (TiDB 6.1 versions)
- [x] v6.0 (TiDB 6.0 versions)
- [x] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)
- [ ] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Review by qqqdan — APPROVED** [review]

File: quick-start-with-tidb.md [hunk 1]
```diff
@@ -412,6 +412,10 @@ TiDB 是一个分布式系统。最基础的 TiDB 测试集群通常由 2 个 Ti
     Input SSH password:
     ```
 
+    > **注意：**
+    >
+    > 如果本部署主机通过密钥进行 SSH 认证，请使用 `-i` 参数指定密钥文件路径。
```
Comment (ran-huang) [inline quick-start-with-tidb.md:417]: 往上移到 L407 的位置？用户读到这里的时候，可能已经执行完了。

File: quick-start-with-tidb.md [hunk 2]
```diff
@@ -404,6 +404,11 @@ TiDB 是一个分布式系统。最基础的 TiDB 测试集群通常由 2 个 Ti
 
     - 参数 `<cluster-name>` 表示设置集群名称
     - 参数 `<tidb-version>` 表示设置集群版本，可以通过 `tiup list tidb` 命令来查看当前支持部署的 TiDB 版本
+    - 参数 -p 表示在连接目标机器时使用密码登录
```
Comment (shichun-0415) [inline quick-start-with-tidb.md:407]:
```suggestion
    - 参数 `-p` 表示在连接目标机器时使用密码登录
```

**Review by ran-huang — APPROVED** [review]
LGTM

File: quick-start-with-tidb.md [hunk 3]
```diff
@@ -404,6 +404,11 @@ TiDB 是一个分布式系统。最基础的 TiDB 测试集群通常由 2 个 Ti
...
+
+        > **注意：**
+        >
+        > 如果本部署主机通过密钥进行 SSH 认证，请使用 `-i` 参数指定密钥文件路径，`-i` 与 `-p` 不可同时使用。
```
Comment (shichun-0415) [inline quick-start-with-tidb.md:411]:
```suggestion
        > 如果主机通过密钥进行 SSH 认证，请使用 `-i` 参数指定密钥文件路径，`-i` 与 `-p` 不可同时使用。
```

**Review by shichun-0415 — COMMENTED** [review]
rest LGTM

**Pull request merged**